test/data/tree-construction-regression/gh40_form_in_template.dat

```diff
@@ -1,13 +0,0 @@
-#data
```

Comment:
What’s the reason this has to be removed? Is it wrong?

Reply:
It was upstreamed in https://github.com/html5lib/html5lib-tests/pull/145. I didn't remove it before because we didn't actually run the upstream html5lib-tests tree construction tests. Now that we do, it is save to remove it.

Reply:
ah okay, makes sense!